man/shutdown.Rd

```diff
@@ -27,3 +27,4 @@ Other main user interface functions: \code{\link{fetchvars}},
   \code{\link{newcore}}, \code{\link{reset}},
   \code{\link{run}}, \code{\link{setvar}}
 }
+\concept{main user interface functions}
```

Comment:
I don't love that the `@family` tag adds these as search keywords.  I picked the family tags to make the text in the `\seealso` make sense (in this case, "See also: Other main user interface functions...".  I wouldn't necessarily want these functions to show up when a user searches for any of these keywords individually.  Is there a way to turn off this behavior, or are we stuck with it?

Reply:
This was Hadley's doing -- https://github.com/klutometis/roxygen/issues/611. Doesn't seem to be configurable in any way. I could go through and remove each one manually, but then everyone working with the latest Roxygen would have to do it every time as well. 😞

Reply:
Yeah, let's not do that.